### ISSUE TITLE
Social icons in view in browser

### ISSUE THREAD

**Issue by pbatroff** [systopia/CiviProxy#38]
Social Icons in the versafix template use relative paths, thus the replacement on the proxy to view the mailing doesn't work. Example URL:

`/sites/default/files/civicrm/persist/mosaico_tpl/versafix-1.1/img/icons/fb-colors-96.png`

This then wont be replaced in the proxy `civiproxy_mend_URLs`, then having an invalid URL. An additional regex fix should fix the problem for the `$target_file` replacement.

**Comment by pbatroff** [comment]
Alternatively a [page hook](https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_pageRun/) could be used in the civiproxy extension on civi side to change the links.

**Comment by pbatroff** [comment]
I created an issue with the mosaico extension [here](https://github.com/veda-consulting/uk.co.vedaconsulting.mosaico/issues/349), ideally this should be fixed in there.

**Comment by jaapjansma** [comment]
I have difficulties with replacing social media icons from the versafix template. The instance is CiviCRM 5.30.1 on Wordpress and Mosaico 2.5.1597918155 

I see that the templates have an absolute url.

**Comment by bjendres** [comment]
@pbatroff is this currently working?

**Comment by pbatroff** [comment]
Yes, it is both working with  the normal packed icons as well as with Icons in custom themes, as long as the theme folder is located somwhere in [civicrm_files]/persist/
